ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -159,19 +157,49 @@ if [[ ! -z "$OLD_CLUSTER" ]]; then
 fi
 
 # Get the openshift version
-version_cmd="rosa list versions --channel-group ${CHANNEL_GROUP} -o json"
-if [[ "$HOSTED_CP" == "true" ]]; then
-  version_cmd="$version_cmd --hosted-cp"
+if [[ -z "${CHANNEL_GROUPS}" ]]; then
```

Comment:
I still think we should keep this as `CHANNEL_GROUP` singular.  The step-registry ref may not need to be changed.